tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- needs to contain all frameworks which src projects wish to restore -->
+    <TargetFramework Condition="'$(UseIntrinsics)' != 'true'">netstandard2.0</TargetFramework>
+    <TargetFrameworks Condition="'$(UseIntrinsics)' == 'true'">netstandard2.0;netcoreapp3.0</TargetFrameworks>
```

Comment:
Why should we have the condition here?  Is there any harm in always resolving for both TFMs?

Reply:
Usually `netcoreapp3.0` causes problems when you are building in VS 2017 where the SDK throws an error saying `I don't support netcoreapp3.0`.

Reply:
I see, makes sense.